lib/features/mailbox/presentation/mailbox_view.dart

```diff
@@ -36,6 +36,22 @@ class MailboxView extends BaseMailboxView {
           children: [
             Expanded(
               child: Column(children: [
+                Obx(() {
+                  final accountId = controller
+                      .mailboxDashBoardController
+                      .accountId
+                      .value;
+
+                  final username = accountId != null
+                    ? controller.mailboxDashBoardController.getOwnEmailAddress()
+                    : '';
```

Comment:
```suggestion
                  final username = controller.mailboxDashBoardController.getOwnEmailAddress();
```

IMO `getOwnEmailAddress` from `mailboxDashBoardController` doesn't depend on `accountId` to call, and always return non-nullable String, so this would suffice.

Reply:
We need an Obs variable to trigger the Obx widget, getOwnEmailAddress it depends on the session and the session always comes with the accountId so we can use the accountId for this.